.env.example

```diff
@@ -1,9 +1,11 @@
 PORT=3000
 NODE_ENV=development
 CREDENTIALS = {"type": "service_account","project_id": "incredible-phantom","private_key_id": "1b5a492efd26393c7edbc6e045709e1d0098c25f","private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDBvxrL4cu9PGqF\neSY59psA7D9D9B/u6GU3NT5p0bObLgPTf+HbzLy+xQNTem9IKLzAzlX9YEVPJh1y\n7Xg4BEXEAK9HepBhUhEjwT8wFYzg1d3qd+VezonhqfjdfA+DwKOjJabe/jOZqIAh\nzA31GMM4gFtLZVhD6gIxULXDuvQN6cMzT8BWqzS1VT7ofQa+UDvAA+ugTVqPKRc1\ngjhCjVZbFReabbc7NRQTAp4tdCgU5vlCOSgCDVsRLf6ufTEIuRHQp8ycFpZtZMGQ\nnTsFm15xHtzV7t4UjGVXnTut4iScFJs6boiI1fPcm9bIE6mWG6gh7B3iCnE5FEVU\nLbTIQ6M1AgMBAAECggEAWnfsAUvcIO73JoAdk29Nja7DE+4IgzxNFuxrymm7Hjjs\nXHQYeNQvRj0kgnRnRiCPktBGYtvzQTQhCCtOuYl1MkLHle7rQCZcDoAMC13kvI/g\n4b0+g3iZ9Z+yr2lgrTrkrifEtdepLJT7z8xCsDHx5o45Ewmtn3jDsVdoqI7Bq0Su\nFLR6EqGJCn638IFd4gTR1fa35JP3up4LZdk/BcDcgdh5Jj7Hxk1JIicanWaaO4/j\nmHMofyf6F8ukljpgcf2LevUBfhcLg2y8CkEx9kcUObSTRS/7l9T9SV0vvEcKY1LV\n7DteD2HIeB3p/InHoANAQMfMcitiQdyRTfurfzXheQKBgQDtNUH0qQQFzgGz7QRq\nnrDpu1T3jHCbCMA+hLW6Ymj9XZtrtrmW2zeMHYr2JpUGSdRGOJihuEGQmZM7UjcD\nc7983SgLrAqBK4aQTPkWU15p2lxv8f45A0ffxY+QuWLZ2wnqg8yCvRvQU+QGvFsp\njYMlU+A4KwjNJqW9zlRT9N8eUwKBgQDRGGrL3q2PeWxAN+Ang5m7xJlsAsOzCfM2\nDNSMeSoAIFFO4E7Nw+pUJdntbCrPH9S34wF47zGwRLpdWmAfQdbJMpqi8loAlzXZ\nq2VmiXB7hG6yEyTsJFpQFCp4IYfJJzlb8hEuNiMcI8s4PgQOP4yuEh/L/+vWQR7y\ngzbXcNG3VwKBgFqUlcUZrYGR8zCD98XHJ4N5l66YIH/zTYYSI1szLShjUHBHbINg\n3stQlLn+bbXjB35xKkfHFrJjEsf6UyusTZke0qXyYmiE+HOuEQgnecwuKFJPNGcN\nC6s1w+beBBNxA05cWgIquK5K+q5fyugJM+VTLDIGrOu6MFxUp/sPbQwlAoGAbcQH\nrGaFbmArt7qdbaDkP3IBqHX7Di4XAzTRD707kz1j8ahHNjW2+d20/bJe9sSbAr2k\ntDu9Ne+edgtPkpkZjB2ImCnpnY3DszUK0j0sU2s+QTVTmLOsxYUQlmfjxHFW5zG4\nWy4IWGlh8XB86I3r6CWTk6ARyjP9bwn67DZJoZkCgYEAu8YhIfEKvVj/X6XWDZYK\ndx6HUQbCjzgX79m0xqugHHe3B5vjNdZbejrEefE2Ok0DfIWrFzhp18PLVRjP0xTB\nnU/wBUZI/1FxBrVeZoXHLYM5DC5mfCyYJn5JHWdmoCxbhWODP3DmI3K3QKEq/xfM\nOoY6y+LH0pS/PhP6E4uyM5s=\n-----END PRIVATE KEY-----\n","client_email": "incredible-phantom@incredible-phantom.iam.gserviceaccount.com","client_id": "108890191225603873059","auth_uri": "https://accounts.google.com/o/oauth2/auth","token_uri": "https://oauth2.googleapis.com/token","auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs","client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/incredible-phantom%40incredible-phantom.iam.gserviceaccount.com"}
-DEV_DATABASE_URL=postgres://postgres:<password>@<host>:5432/phantom-dev
-TEST_DATABASE_URL=postgres://postgres:<password>@<host>:5432/phantom-dev
-DATABASE_URL=postgres://postgres:<password>@<host>:5432/phantom-dev
-JWT_KEY=<jwt key>
-EMAIL=<email>
-PASS=<password>
+DEV_DATABASE_URL=postgres://postgres:pass2db@localhost:5432/phantom-dev
+TEST_DATABASE_URL=postgres://postgres:pass2db@localhost:5432/phantom-test
+DATABASE_URL=postgres://postgres:pass2db@localhost:5432/phantom
+JWT_KEY=helloPhantom
+EMAIL=atlpincredible@gmail.com
+PASS=andela@2020
```

Comment:
you didn't have to change the values for DEV_DATABASE_URL, TEST_DATABASE_URL, DATABASE_URL, JWT_KEY, EMAIL